File: travel.js
function depositMoney() {
    let balance = 0;
    let monthsPassed = 0;
  
    const intervalId = setInterval(() => {
      balance += 1000;
      monthsPassed++;
  
      console.log(`1000 pesos deposited. Current balance: ${balance} pesos.`);
  
      if (monthsPassed === 12) {
        clearInterval(intervalId);
        console.log("Funds have been deposited for 12 months. Program terminated.");
      }
    }, 30 * 24 * 60 * 60 * 1000);
  }
  
  depositMoney();
  

File: travel.py
import time

def deposit_money():
    balance = 0
    months_passed = 0

    while months_passed < 12:
        balance += 1000
        months_passed += 1
        print(f'1000 pesos deposited. Current balance: {balance} pesos.')
        
        time.sleep(30 * 24 * 60 * 60)

    print("Funds have been deposited for 12 months. Program terminated.")

if __name__ == "__main__":
    deposit_money()